pkg/storage/bucket/client.go

```diff
@@ -138,7 +150,7 @@ func (cfg *Config) disableRetries(backend string) error {
 		cfg.Azure.MaxRetries = 1
 	case Swift:
 		cfg.Swift.MaxRetries = 1
-	case Filesystem:
+	case Filesystem, Alibaba, BOS:
```

Comment:
Just noticed: with https://github.com/thanos-io/objstore/pull/147 merged we can now define these for these two no? Also fine to do it on a follow up

Reply:
both bos and oss do not have a any way to configure retries, that's what i gathered from my last search :)